apps/website/docs/cli/savemoney.md

```diff
@@ -0,0 +1,396 @@
+---
+sidebar_position: 2
+title: SaveMoney - Cost Optimization
+description: Analyze cloud resources to identify cost optimization opportunities
+---
+
+The SaveMoney tool helps identify underutilized and unused cloud resources that
+may be costing your organization money. It performs read-only analysis across
+your cloud infrastructure, generating actionable reports to support FinOps
+decisions without modifying any resources.
+
+## Overview
+
+SaveMoney scans your cloud subscriptions using provider APIs and metrics to
+scientifically detect:
+
+- **Inactive Resources** - VMs, storage, and services with minimal usage
+- **Orphaned Resources** - Unattached disks, unused IPs, and dangling network
+  interfaces
+- **Oversized Resources** - Services running on unnecessarily expensive tiers
+- **Misconfigured Resources** - Resources in wrong regions or missing management
+  tags
+
+All analysis is performed in **read-only mode** - the tool never modifies, tags,
+or deletes resources.
+
+## Supported Cloud Providers
+
+### ✅ Azure
+
+Full support for Azure resource analysis with intelligent detection algorithms
+based on Azure Monitor metrics and resource states.
+
+### 🚧 AWS (Planned)
+
+AWS support is planned for future releases with similar capabilities.
+
+## Quick Start
+
+```bash
+# Interactive mode (prompts for configuration)
+npx @pagopa/dx-cli savemoney
+
+# Using configuration file
+npx @pagopa/dx-cli savemoney --config config.json
+
+# With verbose output and JSON format
+npx @pagopa/dx-cli savemoney --config config.json --format json --verbose
+```
+
+## Architecture
```

Comment:
thoughts: It appears this chapter is primarily aimed at tool _developers_ rather than end-users. I suggest moving this information to the tool `README` and keeping only the final user-related details here on the documentation website